File: api/DTOs/Admin/SelDecisionDto.cs
namespace api.DTOs.Admin
{
    public class SelDecisionDto
    {
        public int Id { get; set; }  //
        public int CvRefId { get; set; }
        public int OrderItemId { get; set; }
        public int OrderId { get; set; }
        public int CandidateId { get; set; }
        public int ProfessionId { get; set; }
        public string Gender { get; set; }
        public string SelectedAs { get; set; }  //
        public int Charges { get; set; }            //
        public string CandidateName { get; set; }
        public int ApplicationNo { get; set; }
        public string CustomerName { get; set; }
        public string CategoryRef { get; set; }
        public DateTime ReferredOn { get; set; }
        public DateTime SelectedOn { get; set; }
        public string SelectionStatus { get; set; }
        public int EmploymentId { get; set; }
        //public string RejectionReason { get; set; }
        
    }
}

File: client/src/app/_dtos/admin/selDecisionDto.ts
export interface ISelDecisionDto
{
    id: number;
    checked: boolean;
    cvRefId: number;
    orderItemId: number;
    orderId: number;
    professionId: number;
    candidateId: number;
    candidateName: string;
    applicationNo: number;

    customerName: string;
    categoryRef: string;
    referredOn: Date;
    selectedOn: Date;
    selectionStatus: string;
    //rejectionReason: string;
    
}